src/websocket.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::{anyhow, Context, Result};
+use holochain::conductor::api::{AdminRequest, AdminResponse, AppStatusFilter, InstalledAppInfo};
+
+use holochain_types::dna::AgentPubKey;
+use holochain_websocket::{connect, WebsocketConfig, WebsocketSender};
+use std::sync::Arc;
+
+use log::{debug, info};
+use url::Url;
+
+#[derive(Clone, Debug)]
+pub struct AdminWebsocket {
+    tx: WebsocketSender,
+    agent_key: Option<AgentPubKey>,
```

Comment:
I am getting: `warning: field is never read: "agent_key"`, which seems to be a case. What if we remove this field here and on line 31?